### ISSUE TITLE
Not compatible with Vue 3

### ISSUE THREAD

**Issue by maxacarvalho** [shayneo/vue-fuse#93]
Hello,

I can see that you list Vue 3 as a compatible version but the package.json locks the dependency at Vue 2.x.
Is there any other repo?

**Comment by shayneo** [comment]
It should be compatible, the lib exports the .vue component directly, which uses the composition API in a way that is valid in both Vue 2 and 3.

Please let me know if you have any issues with this

**Comment by maxacarvalho** [comment]
@shayneo if you check your `package.json` you will see:

```
...
"peerDependencies": {    "vue": "^2.6.10"  },
...
```

and

```
"devDependencies": { 
...
  "vue": "^2.6.10",
...
},
```

Also from your `package.json`, I can see `"main": "src/components/VueFuse.vue",`. The Vue component exported by your lib does not make use of the composition API, which is not the problem, since it should work with Vue3 and "legacy" mode.  
I'm just pointing that out since you mentioned the composition API, maybe I'm missing something here, maybe there's a different hidden branch that I should pull the package from, but so far, NPM will not let me install the package on a Vue 3 project and the exported component does not use the composition API.

**Comment by shayneo** [comment]
> @shayneo if you check your `package.json` you will see:
> 
> 
> 
> ```
> 
> ...
> 
> "peerDependencies": {    "vue": "^2.6.10"  },
> 
> ...
> 
> ```
> 
> 
> 
> and
> 
> 
> 
> ```
> 
> "devDependencies": { 
> 
> ...
> 
>   "vue": "^2.6.10",
> 
> ...
> 
> },
> 
> ```
> 
> 
> 
> Also from your `package.json`, I can see `"main": "src/components/VueFuse.vue",`. The Vue component exported by your lib does not make use of the composition API, which is not the problem, since it should work with Vue3 and "legacy" mode.  
> 
> I'm just pointing that out since you mentioned the composition API, maybe I'm missing something here, maybe there's a different hidden branch that I should pull the package from, but so far, NPM will not let me install the package on a Vue 3 project and the exported component does not use the composition API.

Whoops meant to say the options api.

There is no other branch, it's intentionally exporting the .vue file which will work in 2 and 3.

Would you mind sending a link to a repo that reproduces? He happy to take a look, might just need a tweak to the peer dependency requirement.

**Comment by maxacarvalho** [comment]
@shayneo 

You don't need a repo to reproduce the error, just do the following:

1. Initiate an empty project `npm init`
2. Install Vue3 `npm install vue@next`
3. Try to install your package

<img width="722" alt="CleanShot 2021-06-19 at 10 54 24@2x" src="https://user-images.githubusercontent.com/45166618/122638573-bad03a80-d0ec-11eb-9f41-d07e6d5f6f5e.png">

**Comment by shayneo** [comment]
thanks @maxacarvalho just published 3.0.1 which should fix: https://github.com/shayneo/vue-fuse/pull/94
